content/modules/bpm/pages/index.adoc

```diff
@@ -1,20 +1,24 @@
 = BPM
 
-include::ROOT:partial$outdated-ui-warning.adoc[]
+BPM add-on provides functionality for modeling and running business processes.
 
-BPM add-on is used for modeling and running business processes. Processes are described using BPMN 2.0 notation. The add-on uses https://flowable.com/open-source/[Flowable^] as a business process engine and https://bpmn.io/[bpmn-js^] rendering toolkit for process modeling.
+[[license]]
```

Comment:
Remove this and add the same information about license/trial as in other commercial add-ons, e.g. Maps (in the Installation section).

Reply:
Done